src/pages/projectSettings/HeaderButtons.tsx

```diff
@@ -126,22 +131,23 @@ export const HeaderButtons: React.VFC<Props> = ({ id, projectType, tab }) => {
       >
         Save Changes on Page
       </Button>
-      {projectType === ProjectType.AttachedProject && (
-        <>
-          <Button
-            onClick={() => setDefaultModalOpen(true)}
-            data-cy="default-to-repo-button"
-          >
-            Default to Repo on Page
-          </Button>
-          <DefaultSectionToRepoModal
-            handleClose={() => setDefaultModalOpen(false)}
-            open={defaultModalOpen}
-            projectId={id}
-            section={mapRouteToSection[tab]}
-          />
-        </>
-      )}
+      {projectType === ProjectType.AttachedProject &&
+        !defaultToRepoDisabled.has(tab) && (
```

Comment:
I think it might be slightly more readable if we stored this in a variable above like

`const canDefaultToRepo = !defaultToRepoDisabled.has(tab)`

and then did 
```
{ projectType === ProjectType.AttachedProject && canDefaultToRepo && ... }
```